src/main/java/com/thevoxelbox/voxelsniper/config/VoxelSniperConfigLoader.java

```diff
@@ -1,19 +1,41 @@
 package com.thevoxelbox.voxelsniper.config;
 
+import com.sk89q.worldedit.world.block.BlockType;
+import com.sk89q.worldedit.world.block.BlockTypes;
+import org.bukkit.configuration.ConfigurationSection;
 import org.bukkit.configuration.file.FileConfiguration;
 
+import java.util.HashMap;
 import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
 
 /**
  * Configuration storage defining global configurations for VoxelSniper.
  */
 public class VoxelSniperConfigLoader {
 
     private static final String MESSAGE_ON_LOGIN_ENABLED = "message-on-login-enabled";
+    private static final String DEFAULT_BLOCK_MATERIAL = "default-block-material";
+    private static final String DEFAULT_REPLACE_BLOCK_MATERIAL = "default-replace-block-material";
+    private static final String DEFAULT_BRUSH_SIZE = "default-brush-size";
     private static final String LITESNIPER_MAX_BRUSH_SIZE = "litesniper-max-brush-size";
     private static final String LITESNIPER_RESTRICTED_MATERIALS = "litesniper-restricted-materials";
+    private static final String BRUSH_SIZE_WARNING_THRESHOLD = "brush-size-warning-threshold";
+    private static final String DEFAULT_VOXEL_HEIGHT = "default-voxel-height";
+    private static final String DEFAULT_CYLINDER_CENTER = "default-cylinder-center";
```

Comment:
Where are the added entries in the config?

Reply:
```yaml
message-on-login-enabled: true
default-block-material: AIR
default-replace-block-material: AIR
default-brush-size: 3
litesniper-max-brush-size: 5
litesniper-restricted-materials: [BARRIER, BEDROCK]
brush-size-warning-threshold: 20
default-voxel-height: 1
default-cylinder-center: 0
```